Genereate and implement Maps Timezone V1

jecmenicanikola opened the pull request:
This PR contains:

- Generated code for Maps Timezone from preview/1.0/timezone.json
- Implementation of custom code
- Add samples
- Add tests

Fixes https://github.com/Azure/azure-sdk-for-net/issues/42558

jecmenicanikola committed:
Genereate and implement Maps Timezone V1

jecmenicanikola committed:
Add tests and samples

jecmenicanikola committed:
Set correct version and fix cspell

jecmenicanikola committed:
Fix test client

jecmenicanikola committed:
Use GeoPosition for coordinates

jecmenicanikola committed:
Update asset repo

jecmenicanikola committed:
Run ExportAPI script

jecmenicanikola committed:
Add timezone in general README.md file

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.Maps.Timezone](https://apiview.dev/Assemblies/Review/be261ccf752b4a2ba9c2b858fcccf100?revisionId=b399ae11ecc94a15a756dd3a98bd26a7)

jecmenicanikola committed:
Update readme file

jecmenicanikola commented:
/azp run prepare-pipelines

jecmenicanikola commented:
/azp run net - maps - ci

dubiety commented:
Hi @jecmenicanikola , we need to change **Timezone** to **TimeZone** or **Time Zone**. I've confirmed they're 2 words. Sorry I didn't remind you previously.

dubiety commented on sdk/maps/Azure.Maps.Timezone/samples/MapsTimezoneSamples.md:
```diff
@@ -0,0 +1,50 @@
+## Examples
+
+You can familiarize yourself with different APIs using our [samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Timezone/samples). 
```
We don't need to add this sentence here because `/samples/README.md` also point to this page.
We also need to guide users what namespace they should import. Please refer to [Rendering sample](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/maps/Azure.Maps.Rendering/samples/MapsRenderingTilesImageriesImagesSamples.md)

dubiety commented on sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClient.cs:
```diff
@@ -0,0 +1,415 @@
+        }
+
+        /// <summary>
+        /// Use to get the list of Windows Time Zone IDs.
```
lower case "time zone"

dubiety commented on sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClient.cs:
```diff
@@ -0,0 +1,415 @@
+        /// </summary>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <remarks>
+        /// The `Get IANA Time Zones` API is an HTTP `GET` request that returns a full list of Internet Assigned Numbers Authority (IANA) time zone IDs. Updates to the IANA service are reflected in the system within one day.
```
use `<c>xxx</c>` to replace backticks

dubiety replied:
Same for L#304, L#326, L#348, L#373, L#398

dubiety commented on sdk/maps/Azure.Maps.Timezone/src/Models/Options/TimezoneBaseOptions.cs:
```diff
@@ -0,0 +1,26 @@
+
+namespace Azure.Maps.Timezone.Models.Options
+{
+    /// <summary> Options. </summary>
```
```suggestion
/// <summary> Options for TimeZone Client. </summary>
```

dubiety commented on sdk/maps/Azure.Maps.Timezone/src/Models/Options/TimezoneBaseOptions.cs:
```diff
@@ -0,0 +1,26 @@
+        /// <summary> Specifies the language code in which the timezone names should be returned. Please refer to <see href="https://docs.microsoft.com/azure/azure-maps/supported-languages">Supported Languages</see> </summary>
+        public string AcceptLanguage { get; set; }
+
+        /// <summary> Alternatively, use alias "o". Options available for types of information returned in the result. </summary>
```
Do we really have the alias ("o", "s", "tf", "ty") for users? How do they use in C# SDK?

jsquire reviewed (dismissed):
@jecmenicanikola :

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

christothes committed:
Remove duplicate SYSTEM_ACCESSTOKEN (#45004)

alrod committed:
Fixing target base scale instance concurrency for queues (#44661)

* Fixing target base scale instance concurrency for queues

* New commit

* Adding changelog notes

* Fix

---------

Co-authored-by: Amanda Nguyen <amnguye@microsoft.com>

memontic-ms committed:
[Azure.Communication.Messages] Update README (#44987)

* Update Azure.Communication.Messages README

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.Exporter (#45037)

azure-sdk committed:
Increment version for storage releases (#45039)

weshaggard committed:
Update Preview Test Sub to use new connection (#45035)

In order to get both deployment and clean-up working we need
to have a connection that points at the right subscription. Today azure-sdk-tests points to the name Test Sub but I created azure-sdk-tests-preview connection which points to the Preview Test Sub.

This will fix the clean-up step which is currently failing for things deployed in Preview.

amnguye committed:
Bumped Storage, Blobs, and Queues package dependency version (#45016)

amnguye committed:
[Storage] [DataMovement] Improve checking on part completion for pause/stop/completion status (#45038)

* WIP

* Make CI happy

* Revert "Make CI happy"

This reverts commit a7e80cfc27850b62075ea05c22475080ed79b7f4.

* WIP

* Cleanup

* More cleanup

amnguye committed:
Updated Changelog for DataMovement July Release (#45040)

Commit:
[AzureMonitorDistro] prep distro 1.3.0-beta.1 (#44991)

* prep distro 1.3.0-beta.1

* change to package reference

amnguye committed:
Changed DataMovement Blobs and File Shares to use package dependency, Added File Shares to packages.data.props (#45042)

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.AspNetCore (#45043)

azure-sdk committed:
Increment version for storage releases (#45045)

amnguye committed:
[Storage] [Webjobs Extension] Updated changelog to prepare for WebJobs Storage Extension release (#45041)

azure-sdk committed:
Increment version for storage releases (#45054)

subhra231 committed:
Update Changelog and readme files to include --prerelease (#45047)

* Update Changelog File to include release date.

* Update Readme to add --prerelease option

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#45056)

ShivangiReja committed:
[Search] GA Features for API Version 2024-07-01 (#44485)

* [Search] GA Features for API Version 2024-07-01

Commit:
[AzureMonitor] add Log Scopes example to Readme (#44749)

* add Log Scopes example to Readme

* pr feedback

* add AzureMonitor to example code

* pr feedback

* Update sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

Co-authored-by: Cijo Thomas <cithomas@microsoft.com>

* add another test

* pr feedback

* pr feedback

* Update sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

Co-authored-by: Rajkumar Rangaraj <rajrang@microsoft.com>

* fix

* pr feedback

---------

Co-authored-by: Cijo Thomas <cithomas@microsoft.com>
Co-authored-by: Rajkumar Rangaraj <rajrang@microsoft.com>

ArthurMa1978 committed:
Prepare release for mysql 1.1.0 (#45051)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.MySql (#45067)

azure-sdk committed:
Increment package version after release of Azure.Search.Documents (#45062)

HarveyLink committed:
[AppService] Bump api version to 2023-12-01 (#43706)

m-redding committed:
[Azure.Messaging.EventHubs] Prepare for release (#45061)

* [Event Hubs] Prepare for release

* update version number to be greater than the beta

* update version

scottaddie committed:
Remove duplicate credential instantiation line in ACS Email README (#45076)

jsquire committed:
[Service Bus] July 2024 release prep (#45075)

* Preparation for the July 2024 release.

JimSuplizio committed:
Nightly docs publish should honor skipPublishDocMs (#45077)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventHubs (#45079)

jsquire committed:
[Service Bus] Change log typo (#45082)

The focus of these changes is to fix a typo that collapsed bullet points into one item.

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8602 (#45019)

* add additional argument to Save-Package-Properties to allow for usage in pull request context

---------

Co-authored-by: Scott Beddall (from Dev Box) <scbedd@microsoft.com>
Co-authored-by: Scott Beddall <45376673+scbedd@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

m-redding committed:
[Azure.Messaging.EventHubs.Processor] Prepare for release (#45063)

* updates

* version

* updates

JonathanCrd committed:
[Azure.Data.Tables] Update version for release (#45058)

* Update release version in Changelog and project

* Update release date

amnguye committed:
[Storage] [Tests] Fix OAuth Storage Samples (#45083)

* Fix storage samples

* Update snippets

azure-sdk committed:
Increment version for eventhub releases (#45085)

* Increment package version after release of Azure.Messaging.EventHubs.Processor

* Update Azure.Messaging.EventHubs.Processor.csproj

* Update CHANGELOG.md

---------

Co-authored-by: Madalyn Redding <66138537+m-redding@users.noreply.github.com>

scottaddie committed:
Reformat DefaultAzureCredential flow chart (#45086)

rajuanitha88 committed:
Using federated auth for ACS tests (#44886)

* Using federated auth for ACS tests

* Removing usage of secrets for Shortcodes

* Removing PPE and INT Live tests

* Addressing comments to maintain consistency across languages

seanmcc-msft committed:
Updated Storage codeowners (#45088)

christothes committed:
Add exception information for all credential classes (#44633)

azure-sdk committed:
Increment package version after release of Azure.Messaging.ServiceBus (#45087)

ameyaiam3 committed:
Adding a new field (#45012)

yaotongms committed:
Release preview version 2024-05-20 of HybridCompute (#44654)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240719.2 (#45097)

* Update Generator Version 3.0.0-beta.20240719.2

* Update SDK codes mi_pu_4

christothes committed:
Update DefaultAzureCredential, InteractiveBrowserCredential, and DeviceCodeCredential docs (#45009)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.HybridCompute (#45096)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.GuestConfiguration (#45094)

JimSuplizio committed:
Unset the SDKType environment variable for API Docs Gen (#45080)

* Unset the SDKType environment variable for API Docs Gen

* Unsetting didn't work, try a different approach

* set /p:BuildInParallel=false on the build commands

* for the want of a quote, the kingdom fell

* change the mgmt build commands, if those work the if statements will be consolidated

* Remove libtype, everything is Taco Bell

azure-sdk committed:
Add Azure Health Data Service product slug (#45107)

Co-authored-by: Scott Addie <scott.addie@microsoft.com>

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8650 (#45057)

* missed a condition passthrough

---------

Co-authored-by: Scott Beddall <scbedd@microsoft.com>
Co-authored-by: Scott Beddall <45376673+scbedd@users.noreply.github.com>

annelo-msft committed:
SCM: Add support for Retry-After header to default ClientRetryPolicy (#45078)

* Add support for Retry-After header to default ClientRetryPolicy

* Add tests

* updates

* nits

* support http-date format as well

* update CHANGELOG

* add header collection to mock

Y-Sindo committed:
Upgrade Web PubSub data-plane REST API to 2024-01-01 (#44699)

1. Add method overloads to support specifying client type when generating client URI
1. Add Convenient methods to support AddConnectionsToGroups and RemoveConnectionsFromGroups.
---------

Co-authored-by: Siyuan <87063252+xingsy97@users.noreply.github.com>

ArthurMa1978 committed:
Remove all Track 1 mgmt. plane SDK code (#45093)

tejasm-microsoft committed:
Group Quota SDK  (#44180)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Quota (#45123)

christothes committed:
revert removal of warnaserror (#44961)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20240722.4 (#45122)

GrahamMThomas committed:
Initial Creation of Azure.Health.Deidentification Dataplane SDK (#43913)

Adds Azure.Health.Deidentification to the .NET SDK repo

---------

Co-authored-by: Graham Thomas <gthomas@microsoft.com>

GrahamMThomas committed:
Update ci.yml on Azure.Health.Deidentification (#45130)

pharring committed:
Reduce memory allocation in EventSourceEventFormatting (#43947)

JonathanCrd committed:
Adjust release date (#45132)

* Adjust release date

* Fix CI

azure-sdk committed:
Increment package version after release of Azure.Data.Tables (#45137)

ahsonkhan committed:
Update ClientAssertionCredential doc comments to fix copy/paste typos (#44940)

ArthurMa1978 committed:
Remove the SignalR Track 1 code (#45142)

ArthurMa1978 committed:
Clean up Track 1 mgmt. plane related pipeline settings (#45121)

seanmcc-msft committed:
Modified Azure.Storage.sln to pass build analyze step (#45138)

jsquire committed:
[Smoke Tests] (#45148)

Exclude the unbranded OpenAI library from smoke testing.

azure-sdk committed:
Add remaining preview/sov cloud sub configs to common dir (#45159)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

christothes committed:
Use MSAL for managed identity credential implementation (#45084)

jecmenicanikola committed:
Change Timezone to TimeZone

jecmenicanikola committed:
Change Timezone to TimeZone

jorgerangel-msft committed:
test: enable filter by tags tests (#45158)

christothes committed:
Prepare Identity for Jul release (#45162)

jecmenicanikola committed:
Update test namspaces

jecmenicanikola committed:
Update test namspaces

jecmenicanikola committed:
Fix solution projects

jecmenicanikola committed:
Fix solution projects

jecmenicanikola committed:
Change ci.yml file

jecmenicanikola committed:
Change ci.yml file

jecmenicanikola committed:
Fix assembly info

jecmenicanikola committed:
Fix assembly info

GrahamMThomas committed:
Update CHANGELOG.md (#45167)

Commit:
[AzureMonitor] fix flaky tests manipulating Environment Variables (#44968)

* fix flaky test

* refactor DefaultPlatform to only cache defined environment variables

* refactor

* revert change

* more test changes

* more changes

* cleanup

* cleanup

* cleanup

* fix typo

* pr feedback

* fix bugs in tests

* cleanup

* add method overload

* add method overload

* cleanup

* update comment

seanmcc-msft committed:
Fixed string to sign sorting (#45105)

badeamarjieh committed:
Introduce Microsoft Fabric SDK (#44887)

* first commit

* fix tests

* fix tests and Delete operation

* update generated SDK from updated swagger

* fix comments

* fix hdinsiht ci mgmt

* fix renaming models by using rename-mapping

* fix comment

* fix tests.

* .

* bump the swagger to latest

* generate from typespec

* regen from swagger

* generate from typespec

* update

* fix test cases

* fix recordings

* update commit id

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240724.2 (#45170)

raych1 committed:
Uniform logging for sdk automation (#45141)

* Uniform logging for sdk automation
-added logging prefix and next steps
-filter logging in sdk automation

* Update eng/scripts/Invoke-GenerateAndBuildV2.ps1

Co-authored-by: Crystal YU <chunyu@microsoft.com>

* Update eng/scripts/automation/GenerateAndBuildLib.ps1

Co-authored-by: Crystal YU <chunyu@microsoft.com>

* Added guidance to FAQ and exitcode

* Update logging format

* Use write-host for better format

* Added packagename to the output

---------

Co-authored-by: Crystal YU <chunyu@microsoft.com>

azure-sdk committed:
Increment package version after release of Azure.Identity (#45168)

dubiety committed:
Update Azure Maps SDK code owners (#44973)

jecmenicanikola committed:
Rename all files

jecmenicanikola committed:
Rename all files

seanmcc-msft committed:
Prep for Storage GA patch release (#45169)

dubiety commented on sdk/maps/Azure.Maps.TimeZone/README.md:
```diff
@@ -0,0 +1,199 @@
+MapsAccountSasContent sasContent = new MapsAccountSasContent(MapsSigningKey.PrimaryKey, principalId, maxRatePerSecond, start, expiry);
+Response<MapsAccountSasToken> sas = mapsAccount.GetSas(sasContent);
+
+// Create a TimezoneClient that will authenticate via SAS token
```
```suggestion
// Create a TimeZoneClient that will authenticate via SAS token
```

dubiety commented on sdk/maps/Azure.Maps.TimeZone/src/MapsTimeZoneClientOptions.cs:
```diff
@@ -0,0 +1,42 @@
+        public Uri Endpoint { get; set; }
+
+        /// <summary> Initializes new instance of MapsTimezoneClientOptions. </summary>
+        /// <param name="version"> Azure Maps Timezone API version. </param>
```
`TimeZone` for Line 29 and 30

nickliu-msft committed:
Fixed changefeed Auth Sample test (#45164)

ArcturusZhang committed:
Add pipelines for fabric (#45171)

* add a random change to trigger the pipeline

* Update CHANGELOG.md

* regen

---------

Co-authored-by: Arthur Ma <mars.arthur@263.net>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240725.1 (#45180)

* Update Generator Version 3.0.0-beta.20240725.1

* Update SDK codes he_me_3

* Update SDK codes de_he_2

* Update SDK codes me_pu_4

* Update SDK codes co_de_1

* fix issues

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

amnguye committed:
Add Storage Code Owner Nick (#45165)

azure-sdk committed:
Increment version for storage releases (#45181)

jsquire committed:
[Functions] Prevent connection name mistake (#45185)

* [Functions] Prevent connection name mistake

The focus of these changes is to guard against a
common error made when configuring a trigger.  With
these changes, if the trigger's connection property
is set to a valid connection string instead of an
informational name, the mistake will be detected
and sensitive information will be redacted from the
error message to avoid accidental capture in logs
and similar mechanisms.

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

* Update sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubClientFactory.cs

Co-authored-by: Madalyn Redding <66138537+m-redding@users.noreply.github.com>

* Update sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubClientFactory.cs

Co-authored-by: Madalyn Redding <66138537+m-redding@users.noreply.github.com>

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs

---------

Co-authored-by: Madalyn Redding <66138537+m-redding@users.noreply.github.com>

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.EventHubs (#45187)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.ServiceBus (#45188)

seanmcc-msft committed:
Updated Packages.Data.props to point at latest Storage packages (#45183)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Fabric (#45192)

wangbwn committed:
Data Factory DotNet Sdk Release (#45114)

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.TimeZone/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.TimeZone/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/Azure.Maps.Timezone.sln

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/Azure.Maps.Timezone.sln

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/CHANGELOG.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/CHANGELOG.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/Directory.Build.props

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/Directory.Build.props

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/api/Azure.Maps.Timezone.netstandard2.0.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/api/Azure.Maps.Timezone.netstandard2.0.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/assets.json

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/assets.json

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/samples/MapsTimezoneSamples.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/samples/MapsTimezoneSamples.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/samples/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/samples/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Azure.Maps.Timezone.csproj

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Azure.Maps.Timezone.csproj

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Argument.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Argument.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ChangeTrackingDictionary.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ChangeTrackingDictionary.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ChangeTrackingList.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ChangeTrackingList.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ModelSerializationExtensions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/ModelSerializationExtensions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/MultipartFormDataRequestContent.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/MultipartFormDataRequestContent.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Optional.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Optional.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/RequestContentHelper.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/RequestContentHelper.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Utf8JsonRequestContent.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Internal/Utf8JsonRequestContent.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/MapsTimezoneModelFactory.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/MapsTimezoneModelFactory.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/TimezoneRestClient.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/TimezoneRestClient.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClient.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClient.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClientOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/MapsTimezoneClientOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Models/Options/TimezoneBaseOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Models/Options/TimezoneBaseOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/autorest.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/autorest.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/properties/AssemblyInfo.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/properties/AssemblyInfo.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/TimezoneClientLiveTestsBase.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/TimezoneClientLiveTestsBase.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/TimezoneClientTestEnvironment.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/TimezoneClientTestEnvironment.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Samples/TimezoneClientSamples.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Samples/TimezoneClientSamples.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Samples/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Samples/README.md

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/MapsTimezoneTests.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/MapsTimezoneTests.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Azure.Maps.Timezone.Tests.csproj

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/tests/Azure.Maps.Timezone.Tests.csproj

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/CountryRecord.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/CountryRecord.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/CountryRecord.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/CountryRecord.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorAdditionalInfo.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorAdditionalInfo.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorAdditionalInfo.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorAdditionalInfo.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorDetail.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorDetail.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorDetail.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorDetail.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorResponse.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorResponse.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorResponse.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ErrorResponse.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/IanaId.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/IanaId.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/IanaId.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/IanaId.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/JsonFormat.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/JsonFormat.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ReferenceTime.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ReferenceTime.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ReferenceTime.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/ReferenceTime.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/RepresentativePoint.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/RepresentativePoint.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/RepresentativePoint.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/RepresentativePoint.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimeTransition.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimeTransition.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimeTransition.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimeTransition.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneIanaVersionResult.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneIanaVersionResult.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneIanaVersionResult.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneIanaVersionResult.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneId.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneId.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneId.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneId.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneNames.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneNames.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneNames.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneNames.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneResult.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneResult.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneResult.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneResult.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneOptions.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneWindows.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneWindows.Serialization.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneWindows.cs

jecmenicanikola committed:
Delete sdk/maps/Azure.Maps.Timezone/src/Generated/Models/TimezoneWindows.cs

jecmenicanikola committed:
Add readme file

jecmenicanikola committed:
Add readme file

jecmenicanikola committed:
Fix cspell

jecmenicanikola committed:
Fix cspell

jecmenicanikola committed:
Fix broken links

jecmenicanikola committed:
Fix broken links

jecmenicanikola committed:
Generate asset.json

jecmenicanikola committed:
Generate asset.json

jecmenicanikola committed:
Removed old asset.json

jecmenicanikola committed:
Removed old asset.json

jecmenicanikola committed:
Fix broken links

jecmenicanikola committed:
Fix broken links

azure-sdk committed:
Update Generator Version 3.0.0-beta.20240726.3 (#45205)

jecmenicanikola committed:
Rename some attributes

jecmenicanikola committed:
Rename some attributes

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.DataFactory (#45201)

jecmenicanikola committed:
Make JsonFormat internal

jecmenicanikola committed:
Make JsonFormat internal

jecmenicanikola committed:
Generate Maps Rendering V2 (#44892)

* Generate Maps Rendering V2

* Update version

* Run CodeChecks script

* Fix tests

* Run ExportAPI script

* Add accept filed in options

* Fix tests

* Push new recordings

christothes committed:
use federated auth for k8s MI test (#45186)

jecmenicanikola committed:
Generate code

jecmenicanikola committed:
Merge branch 'Maps-Timezone-V1' of github.com:JaneaSystems/azure-sdk-for-net into Maps-Timezone-V1

jecmenicanikola commented:
Something is wrong with this PR, will create new one
